static/css/add_movie.css

```diff
@@ -58,4 +58,13 @@ iframe#trailer {
 
 div.modal-body p.plot{
     text-indent: 2em;
-}
\ No newline at end of file
+}
+
+span.section-title{
+    font-size: 1.2em;
+    text-decoration: underline;
+}
+
+span.sub-section-title{
+    font-size: .7em;
+}
```

Comment:
Try to use underscores for classes. I do this because bootstrap uses hypens and it makes it easy to tell which classes are mine and which are bootstrap's.